### DISCUSSION TITLE
大佬，b站上有人盗这个项目拿来买，1500一份，就改了个封面

### DISCUSSION THREAD

**Discussion by liesgame** [Yin-Hongwei/vue-spring-music#119] [General]
https://www.bilibili.com/video/BV1ZA411a7FM?share_source=copy_web
![image](https://user-images.githubusercontent.com/64682920/157277930-864d5c13-de8d-4422-a520-6be3120dd29c.png)
![image](https://user-images.githubusercontent.com/64682920/157278063-d699486c-322b-4ba9-8738-eaaed01bb7dd.png)
![image](https://user-images.githubusercontent.com/64682920/157278133-8008b1a5-dbb3-45f5-8d4b-776c94770714.png)

**Comment by Yin-Hongwei** [comment]
谢谢提醒
